nixos/modules/services/misc/ollama.nix

```diff
@@ -3,12 +3,19 @@ let
   inherit (lib) types;
 
   cfg = config.services.ollama;
-  ollamaPackage = cfg.package.override {
+
+  stableOverrides = {
     inherit (cfg) acceleration;
     linuxPackages = config.boot.kernelPackages // {
       nvidia_x11 = config.hardware.nvidia.package;
     };
   };
+  unstableOverrides = {
+    inherit (cfg) acceleration;
+  };
+
+  ollamaPackage = cfg.package.override (previousPackageArgs:
```

Comment:
`prev` really was ok though

Reply:
So did the name not cause you confusion?

Reply:
Yeah it wasn't the name, maybe I just misread something. As for the "20' reading" comment, that was to convince myself that `override` will indeed pass an attrset with the same names as the autocalled function's arguments... it didn't occur to me to read the docs instead